soda/core/soda/execution/data_source.py

```diff
@@ -165,6 +165,12 @@ def column_metadata_catalog_column() -> str:
     # SQL Queries
     ######################
 
+    def sql_get_table(self, table_name: str, limit: int = None) -> str:
+        sql = f"SELECT * \n" f"FROM {table_name.lower()}"
```

Comment:
same comments as in the bigquery data source

Reply:
and also here the same comment about templating as I made earlier would apply.